docs/topics/installing-web-console-on-openshift.adoc

```diff
@@ -41,7 +41,7 @@ endif::[]
 ifdef::ocp-41,ocp-311[]
 .Procedure
 
-. Download the installation archive file from the link:https://developers.redhat.com/products/mta/download[{ProductShortName} download page].
+. Navigate to the  link:{MTADownloadPageURL}[{ProductShortName} Download page] and download {ProductShortName} installation file.
```

Comment:
Remove extra space before link.
There should be a definite article before "... file". Also, it's a good practice to use the same names or as close as possible as the file names on the web.

Change "download {ProductShortName} installation file" to "download the Web Console `Local install & OpenShift` file." Use the appropriate attribute for Web Console.